### ISSUE TITLE
Need apt-get and pdftotext

### ISSUE THREAD

**Issue by Achillesjr** [adhocore/docker-lemp#42]
Hey there, Can we enable the apt-get option into this image, and with that, I can install the library pdftotext ?

**Comment by adhocore** [comment]
apt-get no because it is not ubuntu or debian but alpine. so there is `apk` instead. as for pdftotext if it does not add too much Megabytes to the size of image, i could accept your PR adding it, thanks

**Comment by adhocore** [comment]
btw seems pdftotext is not in alpine registry https://pkgs.alpinelinux.org/

**Comment by Achillesjr** [comment]
Hey Buddy, thank you for your response.

I’m having a hard time without this library into the docker image.

pdftotext is part of the Xpdf software suite. Poppler<https://en.wikipedia.org/wiki/Poppler_(software)>, which is derived from Xpdf, also includes an implementation of pdftotext. On most Linux distributions, pdftotext is included as part of the poppler-utils package.[1]<https://en.wikipedia.org/wiki/Pdftotext#cite_note-1>

Do you think it’s possible to add some way to the docker?

Thank you in advance.

Achilles Jr

From: Jitendra Adhikari ***@***.***>
Date: Sunday, October 29, 2023 at 5:40 AM
To: adhocore/docker-lemp ***@***.***>
Cc: Achillesjr ***@***.***>, Author ***@***.***>
Subject: Re: [adhocore/docker-lemp] Need apt-get and pdftotext (Issue #42)

btw seems pdftotext is not in alpine registry https://pkgs.alpinelinux.org/

**Comment by Achillesjr** [comment]
I found pdftotext at Alpine. This is correct?

https://pkgs.alpinelinux.org/contents?branch=edge&name=poppler-utils&arch=x86_64&repo=main


From: Jitendra Adhikari ***@***.***>
Date: Sunday, October 29, 2023 at 5:40 AM
To: adhocore/docker-lemp ***@***.***>
Cc: Achillesjr ***@***.***>, Author ***@***.***>
Subject: Re: [adhocore/docker-lemp] Need apt-get and pdftotext (Issue #42)

btw seems pdftotext is not in alpine registry https://pkgs.alpinelinux.org/

**Comment by adhocore** [comment]
yea seems like so, i searched for `pdftotext` - unware of `poppler-utils`. my only concern is if it causes image size to grow significantly. 10-20MB is okay
now it is 350-355MB https://hub.docker.com/r/adhocore/lemp/tags

**Comment by Achillesjr** [comment]
Hey there. Thanks again for your help. I don’t know how to request this using a PR. Can you help me through this?

I need this update as soon as possible. I will pay you a coffee, for sure.

Best, Achilles

From: Jitendra Adhikari ***@***.***>
Date: Sunday, October 29, 2023 at 6:25 AM
To: adhocore/docker-lemp ***@***.***>
Cc: Achillesjr ***@***.***>, Author ***@***.***>
Subject: Re: [adhocore/docker-lemp] Need apt-get and pdftotext (Issue #42)

yea seems like so, i searched for pdftotext - unware of poppler-utils. my only concern is if it causes image size to grow significantly. 10-20MB is okay
now it is 350-355MB https://hub.docker.com/r/adhocore/lemp/tags

**Comment by adhocore** [comment]
all merged #48

**Comment by adhocore** [comment]
you may need to pull the latest version of lemp image to use this in your machine:

```sh
docker pull adhocore/lemp:8.2
docker pull adhocore/lemp:8.1
docker pull adhocore/lemp:8.0
```
